src/client/core/index.js.wrapper.mustache

```diff
@@ -0,0 +1,12 @@
+(function () {
+    //NOTE: we should have an ability to initialize scripts with different contexts.
+    // It's necessary for an iframe without src attribute because the scripts are not injected
+    // inside it by hammerhead. So, we wrap all scripts with initialization functions.
```

Comment:
Move this kind of comments in templates to the top of the file. And fix formatting (space before `NOTE:`)